src/models/Network.js

```diff
@@ -1,9 +1,10 @@
 import {Blockchains} from './Blockchains';
 
 export default class Network {
-    constructor(_host = '', _port = 0, blockchain = Blockchains.EOS, chainId = ''){
+    constructor(_host = '', _port = 0, blockchain = Blockchains.EOS, chainId = '', _protocol='http'){
```

Comment:
if this change doesnt affect to many places, I would place the _protocol as first item (and change that everywhere).

Reply:
At first when I was writing this that's what I did. But when I looked at the Ethereum support I got a bit paranoid at the changes I would be making. The Ethereum support doesn't use that parameter and I don't reallyuuuu understand how ethereum works at that level.

There also wasn't any code I could I test against. Ie any working Ethereum example apps.

This line is one such example.
https://github.com/EOSEssentials/Scatter/blob/6c380b2bcc2b8b0907686de3b6a7a9b5d6e550f2/src/plugins/defaults/eth.js#L119

On the other hand if you guys don't think this change will screw or the Ethereum support you are right the protocol make a lot more sense as the first parameter